src/components/NFTPage/CollectCTA.vue

```diff
@@ -43,17 +43,14 @@
           <div class="text-[16px] text-white font-600 text-center mt-[4px] sm:text-left">{{ creatorMessage }}</div>
         </div>
       </div>
-      <ButtonV2
-        preset="secondary"
-        class="flex-shrink-0"
-        :text="ctaButtonText"
-        :is-disabled="!isCollectable"
-        @click="handleClickCTAButton"
-      >
-        <template #prepend>
-          <IconPrice />
-        </template>
-      </ButtonV2>
+      <CollectButton 
+        :button-text="ctaButtonText"
+        :is-collectable="isCollectable"
+        :collect-expiry-time="collectExpiryTime" 
+        :expiry-time-class="expiryTimeClass"
```

Comment:
Isn't this a computed prop of `<CollectButton/>`?

Reply:
Yes, legacy code, should be removed